src/containers/table/Table.js

```diff
@@ -294,8 +308,7 @@ class TableView extends React.Component {
               })}
               </tbody>
             </table>
-            <button className='showAll' type="button" onClick={(e) => this.showMore(e)}>Show more</button>
-            <button className='showAll' type="button" onClick={(e) => this.showAll(e)}>Show all</button>
+            {this.showButton(this.state.showCount, searchResults.length)}
```

Comment:
for future reference, this could be written as a pure component and if needed in a separate file.